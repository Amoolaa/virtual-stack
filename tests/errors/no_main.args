"tests/errors/no_main.x2017"
