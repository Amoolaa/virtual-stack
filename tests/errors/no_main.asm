FUNC LABEL 1
    MOV STK A VAL 4
    PRINT STK A
    CAL VAL 3
    RET
FUNC LABEL 3
    MOV STK A VAL 92
    PRINT STK A
    CAL VAL 2
    RET
FUNC LABEL 2
    MOV REG 0 VAL 1
    PRINT REG 0
    RET
