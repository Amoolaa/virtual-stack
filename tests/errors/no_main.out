Entry point (FUNC LABEL 0) does not exist.
